examples/FIL_streamz_pipeline.ipynb

```diff
@@ -0,0 +1,375 @@
+{
```

Comment:
<p>I know you link to the data, but do you want to include how to download that data and get it into Kafka? Perhaps that outside the scope of this notebook though.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rapidsai/clx/pull/273/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>